mdata/importer/serializable_cwr.go

```diff
@@ -0,0 +1,83 @@
+package importer
+
+import (
+	"bufio"
+	"bytes"
+	"compress/gzip"
+	"fmt"
+	"io"
+	"time"
+
+	"github.com/grafana/metrictank/mdata"
+	"github.com/raintank/schema"
+	"github.com/tinylib/msgp/msgp"
+)
+
+//go:generate msgp
+
+// SerializableChunkWriteRequest is used by the importer utility to send cwrs over the network
+// It does not contain the org id because this is assumed to be defined by the auth mechanism
+type SerializableChunkWriteRequest struct {
```

Comment:
"serializable" is not helpful because pretty much all datastructures are serializable .
I would just stick with the name ChunkWriteRequest, the fact that this is in the importer package already gives a solid clue that this CWR type is specific to the importer.
You can see the same pattern in stdlib, for example https://golang.org/pkg/html/template/#Template and https://golang.org/pkg/text/template/#Template

Reply:
For example the `.Callback` property of the `mdata.ChunkWriteRequest` is not serializable because that's a memory address. But I'm fine either way, I'll rename it back to `ChunkWriteRequest`